planetarium/NineChronicles
# Fix not showing lock object in tooltip

### Description

SSIA

### How to test

1. Click items in inventory or any place.
2. Check item icon of tooltip.

### Related Links

[[Previewnet] 착용 및 사용 불가 아이템의 상세 툴팁 아이콘에 자물쇠 UI가 누락된 현상](https://app.asana.com/0/1133453747809944/1202023403020011/f)

### Required Reviewers

@planetarium/9c-dev 

### Screenshot

![image](https://user-images.githubusercontent.com/48484989/160553146-4dca45e4-eed8-4370-8323-3c87faeaafc4.png)
